cmd/clusterctl/client/cluster/template_test.go

```diff
@@ -307,6 +307,10 @@ func Test_templateClient_GetFromURL(t *testing.T) {
 	path := filepath.Join(tmpDir, "cluster-template.yaml")
 	g.Expect(os.WriteFile(path, []byte(template), 0600)).To(Succeed())
 
+	// redirect stdin
+	os.Stdin, err = os.Open(path)
```

Comment:
Please adda  defer block to reset the value of `os.Stdin`.

Reply:
Did it in a very simplistic way, hope it's enough.

Reply:
I think it is enough.